content/en/network_monitoring/netflow/_index.md

```diff
@@ -79,6 +79,23 @@ From the **Configuration** tab in NetFlow, click **Add Enrichment** to upload th
 
 {{< img src="network_device_monitoring/netflow/new_enrichment.png" alt="The New Enrichment Mapping modal in the Netflow configuration tab" width="80%" >}}
 
+### Reverse DNS private IP enrichment
+
+Enable Reverse DNS private IP enrichment to perform DNS lookups and find the domain name associated with any NetFlow source or destination IP address. When enabled, the Agent conducts reverse DNS lookups on source and/or destination IPs within private address ranges, enriching NetFlow records with the corresponding hostnames.
+
+Add the following to your `datadog.yaml` file to enable Reverse DNS IP enrichment (disabled by [default][7]):
```

Comment:
Would it make sense to have this in the main Configuration section, above?  Primarily so the proper indentation would be clear.